lib/Model/TeamMember.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Website\Model;
+
+use Doctrine\SkeletonMapper\Hydrator\HydratableInterface;
+use Doctrine\SkeletonMapper\Mapping\ClassMetadataInterface;
+use Doctrine\SkeletonMapper\Mapping\LoadMetadataInterface;
+use Doctrine\SkeletonMapper\ObjectManagerInterface;
+use function in_array;
+
+class TeamMember implements HydratableInterface, LoadMetadataInterface
+{
+    /** @var string */
+    private $name;
+
+    /** @var string */
+    private $github;
+
+    /** @var string */
+    private $twitter;
+
+    /** @var string */
+    private $avatarUrl;
+
+    /** @var string */
+    private $website;
+
+    /** @var string */
+    private $location;
+
+    /** @var string[] */
+    private $maintains;
+
+    public static function loadMetadata(ClassMetadataInterface $metadata) : void
+    {
+        $metadata->setIdentifier(['github']);
+    }
+
+    /**
+     * @param mixed[] $teamMember
+     */
+    public function hydrate(array $teamMember, ObjectManagerInterface $objectManager) : void
```

Comment:
Types for the properties should be handled in this method or we can get an invalid object state.